ksql-cli/src/main/java/io/confluent/ksql/cli/Cli.java

```diff
@@ -381,7 +376,10 @@ private void printKsqlResponse(RestResponse<KsqlEntityList> response) throws IOE
       boolean noErrorFromServer = true;
       for (KsqlEntity entity : ksqlEntities) {
         if (entity instanceof ErrorMessageEntity) {
-          terminal.printErrorMessage(((ErrorMessageEntity) entity).getErrorMessage());
+          ErrorMessageEntity errorMsg = (ErrorMessageEntity) entity;
+          terminal.printErrorMessage(errorMsg.getErrorMessage());
+          LOGGER.error(errorMsg.toString());
+          LOGGER.error(errorMsg.getErrorMessage().getStackTrace().toString().replace(", ", ", \n "));
```

Comment:
any reason why don't just log `errorMsg.getErrorMessage().toString()`? it has the stack trace etc

Reply:
oops